pyclesperanto_prototype/_tier1/_add_images_weighted.py

```diff
@@ -4,23 +4,38 @@
 from .._tier0 import plugin_function
 from .._tier0 import Image
 
-@plugin_function(output_creator=create_like)
+@plugin_function
```

Comment:
Has that become the default output creator? Or...?

Reply:
Yes, correct!